src/stemmablenote.ts

```diff
@@ -194,29 +194,27 @@ export abstract class StemmableNote extends Note {
 
   /** Gets the `y` value for the top modifiers at a specific `textLine`. */
   getYForTopText(textLine: number): number {
-    if (!this.stave) throw new Vex.RERR('NoStave', 'No stave attached to this note.');
```

Comment:
Let's keep this so we don't need the `?` below.

Reply:
ok